book_maker/loader/epub_loader.py

```
@@ -458,14 +530,285 @@ def process_item(
+                            p, t_text, self.translation_style, self.single_translate
+                        )
+
+                    with self._progress_lock:
+                        if index % 20 == 0:
+                            self._save_progress()
+
+            if soup:
+                chapter_result["processed_content"] = soup.encode(encoding="utf-8")
+            chapter_result["success"] = True
+
+        except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)